docs/features/sessions/5_attaching-datastores.md

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_position: 1.5
+title: Attaching Datastores
+sidebar_label: Attaching Datastores
+---
+
+:::note: Datastore can only be mounted to a Session at Session creation time:::
+
+---
+description: Mount a datastore before starting the Session
```

Comment:
I think we can remove the description. Not a fan of how it looks on the local deployment personally.